libconcurrent/includes/threadtools.h

```diff
@@ -28,27 +53,67 @@ int synchStartThreadsN(uint32_t nthreads, void *(*func)(void *), uint32_t uthrea
 /// @param nthreads The number of posix threads that StartThreadsN spawned.
 void synchJoinThreadsN(uint32_t nthreads);
 
+///@brief This function sets the default placement policy of threads in machine's processors. 
+/// The thread placement policy could set any of the following:
```

Comment:
Why repeating the description of the policies here?

Reply:
Doxygen will vreate a bit better documentation for the man-pages, without having the user to jump from function to function.